core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/inheritance/multipleinheritance/baseannotated/mixed/childannotated/TestMultipleConstraintProvider.java

```diff
@@ -0,0 +1,30 @@
+package ai.timefold.solver.core.impl.testdata.domain.inheritance.multipleinheritance.baseannotated.mixed.childannotated;
```

Comment:
This package name is so insanely long that I bet this will fail on Windows due to path length limits.